tools/crm_node.c

```diff
@@ -128,7 +128,7 @@ remove_cb(const gchar *option_name, const gchar *optarg, gpointer data, GError *
 
     options.command = 'R';
     options.dangerous_cmd = TRUE;
-    options.target_uname = strdup(optarg);
+    pcmk__copy_or_null(&options.target_uname, optarg);
```

Comment:
sparse init should null target_uname as well even if not explicitly listed. so should be safe.